queue_services/common/src/entity_queue_common/service_utils/service_logger.py

```diff
@@ -47,7 +49,12 @@ def before_breadcrumb(crumb, hint):  # pylint: disable=unused-argument; callback
 )
 logging.getLogger('asyncio').setLevel(logging.DEBUG)
 
-if os.getenv('DISABLE_HTTP_ACCESS_LOGS', 'False').lower() == 'true':
+flag_service = Flags()
+if current_app.extensions['featureFlags'] is None:
+    flag_service.init_app(current_app)
+enable_logging = flag_service.value('http-log-enable')
```

Comment:
Could you grab this flag name from a config value that defaults to this so that we can different queues that share the same ld project?